sdk/core/azure-core-tracing-opentelemetry/CHANGELOG.md

```diff
@@ -6,8 +6,20 @@
 
 ### Breaking Changes
 
+- The `Azure::Core::Tracing::OpenTelemetry::OpenTelemetryProvider` type can only
+be instantiated via a factory method: `OpenTelemetryProvider::Create`:
```

Comment:
I'm not a huge fan of not being able to read all the text in my source code.  I tend to insert line breaks to improve the readability of source code because text scrolling off the right margin tends to disappear.

Reply:
Based on my experience, VS code and VS don't do a particularly good job of line wrapping markdown lines.